src/components/TodoHeader/TodoHeader.tsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { ResponseError } from '../../types/enum';
+import { getTodos, createTodo } from '../../api';
+
+type Props = {
+  todos: Todo[];
+  toggleTodosActive: () => void;
+  setIsShowFooter: (arg: boolean) => void;
+  setRespError: (arg: ResponseError) => void;
+  setTodos: (arg: Todo[]) => void;
+  checkCompletedTodo: (arg: Todo[]) => void;
+  userID: number;
+  isDisable: boolean;
+  setIsDisable: (arg: boolean) => void;
+  setCreatingTodoTitle: (arg: string) => void;
+};
+
+export const TodoHeader: React.FC<Props> = ({
+  todos,
+  toggleTodosActive,
+  setIsShowFooter,
+  setRespError,
+  setTodos,
+  checkCompletedTodo,
+  userID,
```

Comment:
no need to lift a state like this. better to create a method in the App and pass it here